From bcfb39087baadf3aaf96b78f38093e4ed7951df4 Mon Sep 17 00:00:00 2001
From: Fam Zheng <famz@redhat.com>
Date: Fri, 17 Jan 2014 03:07:55 +0100
Subject: [PATCH 12/34] qapi: Introduce enum BlockJobType

RH-Author: Fam Zheng <famz@redhat.com>
Message-id: <1389928083-8921-11-git-send-email-famz@redhat.com>
Patchwork-id: 56772
O-Subject: [RHEL-7 qemu-kvm PATCH 10/18] qapi: Introduce enum BlockJobType
Bugzilla: 1041301
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

This will replace the open coded block job type string for mirror,
commit and backup.

Signed-off-by: Fam Zheng <famz@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry picked from commit 2cb5b22286a7546226d9e9363aaee543fcba6b61)
Signed-off-by: Fam Zheng <famz@redhat.com>
---
 qapi-schema.json | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qapi-schema.json |   18 ++++++++++++++++++
 1 files changed, 18 insertions(+), 0 deletions(-)

diff --git a/qapi-schema.json b/qapi-schema.json
index 761512d..b0bd53c 100644
--- a/qapi-schema.json
+++ b/qapi-schema.json
@@ -1407,6 +1407,24 @@
   'data': ['top', 'full', 'none'] }
 
 ##
+# @BlockJobType:
+#
+# Type of a block job.
+#
+# @commit: block commit job type, see "block-commit"
+#
+# @stream: block stream job type, see "block-stream"
+#
+# @mirror: drive mirror job type, see "drive-mirror"
+#
+# @backup: drive backup job type, see "drive-backup"
+#
+# Since: 1.7
+##
+{ 'enum': 'BlockJobType',
+  'data': ['commit', 'stream', 'mirror', 'backup'] }
+
+##
 # @BlockJobInfo:
 #
 # Information about a long-running block device operation.
-- 
1.7.1

